assets/oag.css

```diff
@@ -198,101 +355,310 @@
+			color: #fff;
+		}
+
+		&[data-report-type='NLA'] {
+			background: var(--oag-nla-color);
+			color: #fff;
+		}
 	}
 
-	.label-primary {
-		background: #E3E0D3;
-		border-color: #908F84;
-		color: #4B5B62;
+	.wb-tabs.vertical {
+		display: flex;
+		height: 100%;
+
+		[role="tablist"] {
+			flex: 0 0 30%;
+			display: block;
+			max-width: 300px;
+
+			li {
+				background: none;
+				border: none;
+				border-bottom: 1px solid #959595;
+				display: block;
+				left: 0;
+				padding: 0;
+				text-align: left;				
+				
+				a {
+					border: none;
+					color: var(--oag-body-color);
+					padding: 1em .25em;
+
+					&:hover {
+						color: var(--oag-primary-color);
+					}
+				}							
+			}
+
+			li.active {
+				border-bottom: 3px solid var(--oag-body-color);
+				background: none;
+
+				a {
+					font-weight: 600;
+
+					&:focus, 
+					&:active,
+					&:hover {
+						border:none!important;
+						color: var(--oag-body-color);
+						outline: none!important;
+					}
+				}				
+			}
+		}
+
+		.tabpanels {
+			border: none;
+			flex: 0 0 70%;
+			height: 100%;
+			padding-left: 3em;
+			overflow-y: auto;
+
+			details[open]{
+				border: none;
+				padding: 15px;
+			}
+		}
 	}
 
+	.oag-card {
+		background: #fff;
+        border-radius: var(--oag-rds);
+		display: flex;
+		flex-direction: column;
+		margin-bottom: 20px;
+		overflow: hidden;
+        padding: 20px;
+		position: relative;
+		transition: all ease .5s;
+
+		& > img:first-child {
+			display: block;
+			margin: -20px -20px 20px -20px; 
+			width: calc(100% + 40px); /* 2 × padding */
+		}
+
+		.label + time {
+			margin-left: 1em;
+		}
+
+		time {
+			font-weight: 700;
+			font-size: 75%;
+		}
+
+		h2, h3, h4 {
+			margin-top: 0;
+
+			a {
+				text-decoration: none;
+			}
+		}
+
+		&.bg-light {
+			box-shadow: var(--oag-shadow);
+		}
+
+		&.oag-heading-as-body {
+			h2, h3, h4 {
+				font-family: var(--oag-body-font);
+				font-size: 1em;
+				font-weight: normal;
+
+				a {
+					color: var(--oag-body-color);
+					text-decoration: none;
+
+					&:hover {
+						color: var(--oag-primary-color);
+					}
+				}
+			}
+		}
+
+		&.bg-light:has(.stretched-link):hover {
+			box-shadow: var(--oag-shadow-lg);
+		}
+	}
+
+	.wb-paginate-pager {
+		.pagination {
+			> li {
+				> button {
+					color: var(--oag-primary-color);
+				}
+
+				&.active {
+					> button {
+						background-color: var(--oag-primary-color);
+						border-color: var(--oag-primary-color);;
+						color: #fff;
+					}
+				}
+			}
+		}
+	}
+
+
+	/**********
+		Utilities
+	**********/
+
 	.text-primary {
 		color: var(--oag-primary-color);
-	}		
-	
-	.lead {
-		font-size: 2em;
-		font-weight: 100;
-		line-height: 1.2;
 	}
 
-	.well {
-		border: 2px solid #908F84;
-	}	
+	.text-white {
+		color: #fff!important;
+	}
+
+	.fnt-hdng {
+		font-family: var(--oag-heading-font);
+	}
 
-	.gc-srvinfo a {
-		color: var(--oag-primary-color);
+	.bg-light {
+		background-color: var(--oag-light-bg-color) !important;
+	}
```

Comment:
It will be nice if those CSS selector order can match the order of the corresponding working example.